### PR TITLE
[TSK-268] utilizing scraped data from pipeline to create a jupyter notebook in kubeflow and doing some basic analysis

### PR DESCRIPTION
**Pull request by sk-ram**

### CODE REVIEW

File: docs/guides/kubeflow/analysing-scraped-data.md [hunk 1]
```diff
@@ -3,6 +3,221 @@ sidebar_label: Analyzing Scraped Data
...
+
+}
+
+```
+## 
+#### Download all of the data sources deemed relevant to a local folder in the Kubeflow Notebook
+
+```
+for source in source_list:
+            download_source_if_relevant(source, job_id, tracer_id, scraped_data_repository, work_dir)
```
[the remark this answers is not in the document]
Comment (ydamit) [reply docs/guides/kubeflow/analysing-scraped-data.md:60]: Also, `work_dir` shows up enough throughout this code that it's probably worth defining it somewhere here or earlier - even if it's just a placeholder they're supposed to fill in.

File: docs/guides/kubeflow/analysing-scraped-data.md [hunk 2]
```diff
@@ -3,6 +3,221 @@ sidebar_label: Analyzing Scraped Data
...
+            tweet_month = row['Month']
+            tweet_day = row['Day']
+            tweet_year = row['Year']
+            tweet_disaster_type = row['Disaster_Type']
+
+            # match sattelite image dates with tweet dates
+            if(int(sat_image_year) == int(tweet_year) and sat_image_month == tweet_month and int(sat_image_day) == int(tweet_day)):
+                matches_found_twitter += 1
+                # save the data
+                data.append([f"tweet about {tweet_disaster_type}", tweet_latitude, tweet_longitude, tweet_title, tweet_tweet, tweet_location ])
```
Comment (ydamit) [inline docs/guides/kubeflow/analysing-scraped-data.md:123]: Check indentation

File: docs/guides/kubeflow/analysing-scraped-data.md [hunk 3]
```diff
@@ -3,6 +3,221 @@ sidebar_label: Analyzing Scraped Data
...
+            tweet_tweet = row['Tweet']
+            tweet_location = row['Extracted_Location']
+            tweet_latitude = row['Resolved_Latitude']
+            tweet_longitude = row['Resolved_Longitude']
+            tweet_month = row['Month']
+            tweet_day = row['Day']
+            tweet_year = row['Year']
+            tweet_disaster_type = row['Disaster_Type']
+
+            # match sattelite image dates with tweet dates
```
Comment (ydamit) [inline docs/guides/kubeflow/analysing-scraped-data.md:119]:
```suggestion
            # match satellite image dates with tweet dates
```

File: docs/guides/kubeflow/analysing-scraped-data.md [hunk 4]
```diff
@@ -3,6 +3,221 @@ sidebar_label: Analyzing Scraped Data
...
+        sentinel_df= pd.read_json(os.path.join(sentinel_dir,wildifre_coords_json_file_path), orient="index")
+        for index, row in sentinel_df.iloc[0:].iterrows():
+            lattitude = row['latitude']
+            longitude = row['longitude']
+            status = row['status']
+                                                       #title tweet location
+            data.append([status, lattitude, longitude, "n/a", "n/a", "n/a" ])
+
+
+# load the sattelite image dates
```
Comment (ydamit) [inline docs/guides/kubeflow/analysing-scraped-data.md:101]:
```suggestion
# load the satellite image dates
```

**Review by ydamit — CHANGES_REQUESTED** [review]
Generally looks close to being ready, but there are a few changes that might be important to make this easier to understand and digest.

**Comment by maany** [comment]
Merging! Looks like @sk-ram has addressed the comments from @ydamit

**Pull request merged**